gcovr/cobertura_xml_generator.py

```diff
@@ -146,6 +163,16 @@ def print_xml_report(covdata, output_file, options):
         package[4] += class_branch_hits
         package[5] += class_branches
 
+        class_functions = 0
+        class_function_hits = 0
+        for function_name in sorted(data.functions):
+            class_functions += 1
+            if data.functions[function_name].count > 0:
+                class_function_hits += 1
```

Comment:
Why sorting the list? Starting with Python 3.7 the insertion order of the dict keys is preserved. Older versions do not save the insertion order and the sort costs only time there.

Reply:
Agree, not relevant here. Sorting will be removed